hub-config/tasks.json

```diff
@@ -616,5 +615,6 @@
                 "end": -3
             }
         }
-    ]
+    ],
+    "derived_task_ids": ["target_end_date"]
```

Comment:
@annakrystalli, I want to confirm that this is correct for this hub? The `target_end_date` is derived from `origin_date` and `horizon`. I also set this to `null` and the validation still passed.

Reply:
Yup that looks right. Also, it passed because all `horizon` and `reference_date` are optional. False positives might show up only if there are required values for those task IDs. It will also be faster if you set it.